dev/MAINTAINING.md

```diff
@@ -82,14 +82,20 @@
       to make sure it runs with the latest release.
     * Check the performance of examples where applicable
     * Update streamly-bytestring
-    * Consider updating other packages e.g. streamly-process, streamly-lz4
+    * Update:
+      * streamly-bytestring
+      * streamly-process
+      * streamly-lz4
```

Comment:
Is there a better way to track this? Some reverse dep search commands and update repositories that are controlled by Composewell.